Dockerfile.minikube.deploy

```diff
@@ -0,0 +1,28 @@
+FROM node:16 AS builder
```

Comment:
Is this Dockerfile specific to minikube? If so why? If not please rename it

Reply:
Renamed it, Thanks for the review